modules/ROOT/pages/migration/4.0.0/index.adoc

```diff
@@ -0,0 +1,1012 @@
+[[migration-guide]]
+:description: This page lists the breaking changes from version 3.0.0 to 4.0.0 and describes how to update.
+= Migration to 4.0.0
+:page-aliases: guides/index.adoc, guides/migration-guide/index.adoc, guides/migration-guide/server.adoc, guides/migration-guide/type-definitions.adoc, guides/migration-guide/mutations.adoc
```

Comment:
I'm thinking these page-aliases should be removed in favor to the new migration guide to 5.0.0

```suggestion
```